src/routes/blog/index.svelte

```diff
@@ -72,24 +72,30 @@
     }
   }
 
+  @media (max-width: 640px) {
+    .width-side {
+      display: none;
+    }
```

Comment:
```suggestion
    }
      .width-main {
      width: 100% !important;
    }
```
on mobile the blog articles should take in full screen real estate

Reply:
Done